webapp/src/components/boardsSwitcher/boardsSwitcher.tsx

```diff
@@ -40,12 +40,21 @@ const BoardsSwitcher = (props: Props): JSX.Element => {
         }
     }
 
+    const handleEscKeyPress = (e: KeyboardEvent) => {
+        if (Utils.isKeyPressed(e, Constants.keyCodes.ESC)) {
+            e.preventDefault()
+            setShowSwitcher(false)
+        }
+    }
+
     useEffect(() => {
         document.addEventListener('keydown', handleQuickSwitchKeyPress)
+        document.addEventListener('keydown', handleEscKeyPress)
```

Comment:
Is there any reason to not use the react-hotkeys-hook library? I think is more straight forward.

Reply:
IIRC I tried that and similar to the Cmd/Ctrl + K above, that library has difficulty with shortcuts that are native to the browser/OS.

Reply:
Yep, verified again locally and it doesn't work with hotkeys because of being focused on an input field.